server/chat-commands/info.js

```diff
@@ -2339,6 +2339,19 @@ const commands = {
 	},
 	pickrandomhelp: [`/pick [option], [option], ... - Randomly selects an item from a list containing 2 or more elements.`],
 
+	'!shuffle': true,
+	shuffle(target, room, user) {
+		if (!target || !target.includes(',')) return this.parse('/help shuffle');
+		const args = target.split(',');
+		if (!this.runBroadcast(true)) return false;
+		if (args.length < 3) return this.parse(`${this.cmdToken}pick ${target}`);
```

Comment:
```suggestion
```

This is completely unnecessary. What advantage does `/pick` give for lists of length 2, besides confusing users?